src/Freya.Pipeline/Integration.fs

```diff
@@ -18,7 +18,7 @@
 //
 //----------------------------------------------------------------------------
 
-module Freya.Integration
+namespace Freya.Integration
```

Comment:
I think everywhere else is modules all the way down - is there a specific reason for the use of namespaces here?